sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Create-application-offering-change-requests.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE sims.application_offering_change_requests (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  change_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
+  active_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
```

Comment:
Should we keep the `active_offering_id` in the table or just use the one from the current assessment?

Reply:
mm, that makes sense. I was following the DB structure from the analysis ticket. @andrepestana-aot @dheepak-aot @guru-aot thoughts?

Reply:
I think we already had a discussion, the thought of having the oactive offering id is when we create a componenet which shows both the offeringside byside, the requirement for it would be pass the 2 offering id's and it should render. Yes we can use the current_assessment, but again do we need to go through the application-> current_assessment->offering_id every time? I am okay with not having the offering id explicitly too, but looking at the row in the table can give the active and changed offering directly. Just my thoughts

Reply:
We do not build the DB and API based on how the UI will work, right?
Yes @guru-aot I remember the conversation and I did not remember the output of it.
My point goes beyond saving the current offering id or not. What happens if between the request and the Ministry approval the application went through another reassessment? The real concern is that we would be copying the offering id from the current assessment into this table only to make it better for the UI.

Reply:
I get your point. Does a reassessment happens inbetween? Is that a valid scenario? IF yes, dont we need to stop the reassessment from happening?

Reply:
@andrewsignori-aot  @guru I read through the conversations, and I strongly feel that we need to we need to have `active_offering_id`  here for the sake of accuracy of the application_offering_change_record.

Assume the following scenario without `active_offering_id`

An application with applicationId 100 and offeringId 8

-> applicationId - 100, requested_offering_id-10, status-approved

The application might go through a re-assessment and get associated to an offering 12 later.

If that is the case, then this application offering change record data will say that the offerring change happened from offering 12 to offering 10 which is not accurate. I am talking from the data perspective.

Reply:
keeping as it is and as discussed will confirm with @JasonCTang and will update in next PR